generator-prisma-client/react-router-starter-cloudflare-workerd/worker-configuration.d.ts

```
@@ -6719,6 +6797,19 @@
     interface Env {
     }
 }
+declare module 'cloudflare:node' {
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)